scripts/build-tools.sh

```diff
@@ -5,6 +5,9 @@
 # fail immediately on any errors
 set -e
 
+SOF_TOP=$(cd "$(dirname "$0")/.." && pwd)
+export ALSA_CONFIG_DIR=${SOF_TOP}/tools/topology/topology2
```

Comment:
```suggestion
export ALSA_CONFIG_DIR="${SOF_TOP}"/tools/topology/topology2
```

Use shellcheck. Shellcheck saves lives.